lib/datetime_helper/serialisers/active_model_serialiser_helper.rb

```diff
@@ -11,7 +11,7 @@ def enforce_zulu_time(attribute)
     # Enforces that a Date is serialized as iso8601 format
     def enforce_iso8601_date(attribute)
       define_method(attribute) do
-        object.send(attribute).iso8601 unless object.send(attribute).nil?
+        object.send(attribute).strftime('%F') unless object.send(attribute).nil?
```

Comment:
No, they don't. WIth a date-time object it produces the time as well as the date. For instance, you will get `2015-05-18T22:40:52Z` rather than `2015-05-18`. Not what was desired.